pkg/api/testing/fuzzer.go

```diff
@@ -357,6 +357,11 @@ func FuzzerFor(t *testing.T, version unversioned.GroupVersion, src rand.Source)
 		},
 		func(pvc *api.PersistentVolumeClaim, c fuzz.Continue) {
 			c.FuzzNoCustom(pvc) // fuzz self without calling this function again
+			pvc.Spec.Selector = &unversioned.LabelSelector{
```

Comment:
It should be as easy as adding a function like this? Then you'll test it everywhere it appears.

```go
...,
func(uls *unversioned.LabelSelector, c fuzz.Continue) {
  uls.MatchLabels = map[string]string{
    "testlabelkey": "testlabelval",
  }
},
...
```

Are you sure it's not tested now? I'm pretty sure that the fuzzer recurses into map types.